runners/cromwell_on_google/wdl_runner/test-wdl/basic.papi.us.options.json

```diff
@@ -1,6 +1,6 @@
 {
   "default_runtime_attributes": {
-    "zones": "us-central1-a us-central1-b us-central1-c us-central1-f us-east1-b us-east1-c us-east1-d us-west1-a us-west1-b" 
+    "zones": "us-central1" 
```

Comment:
Sorry my comment was misleading.
As far as I know, Cromwell doesn't support specification of a region here.
You would need to list the us-central1 zones explicitly:

    "zones": "us-central1-a us-central1-b us-central1-c us-central1-f"

Reply:
ah that's disappointing, I thought you were giving me a nifty trick there ;)